os/drivers/gpio/gpio.c

```diff
@@ -289,55 +291,83 @@ static void gpio_enable(FAR struct gpio_upperhalf_s *priv)
 }
 #endif
 
-static int gpio_enable_interrupt(FAR struct gpio_upperhalf_s *priv, unsigned long arg)
+static void gpio_iotbus_interrupt(FAR struct gpio_upperhalf_s *priv)
```

Comment:
Driver should work alone, so it is not good naming.(You may know concept of modularization)
Without iotbus f/w, some module can use this, because it called by generic ioctl.
So please remove all name related to iotbus from driver.
For example, gpio_interrupt_handler, gpio_enable_interrupt.......